docs/development.md

```diff
@@ -55,7 +55,7 @@ To run your app in development, you will need to configure a GitHub App to deliv
     - **Webhook URL**: Use your `WEBHOOK_PROXY_URL` from the previous step.
     - **Webhook Secret:** `development`
     - **Permissions & events** is located lower down the page and will depend on what data you want your app to have access to. Note: if, for example, you only enable issue events, you will not be able to listen on pull request webhooks with your app. However, for development we recommend enabling everything.
-1. Download the private key and move it to your project's directory.
+1. Download the private key and move it to your project's directory. It will get picked up by Probot automatically. If you have several `*.pem` files in the project's directory or want to place it somewhere else, edit `.env` and set `PRIVATE_KEY_PATH`.
```

Comment:
> If you have several `*.pem` files in the project's directory or want to place it somewhere else, edit `.env` and set `PRIVATE_KEY_PATH`.

Most people reading this doc will be creating their very first probot app, so this is irrelevant to them. 
I like the idea of documenting this, but I'm not sure this is the right place for it. The error may be sufficient since someone that copies a second private key into the directory will likely see that before they read any docs about it.

Reply:
I agree that this should be more concise, so I'm fine with moving this somewhere else in documentation or even removing. Although I would keep this instruction alike with the other steps, just set the env var explicitly and don't rely on magic:

> Download the private key to your project's directory and set `PRIVATE_KEY_PATH` in `.env`.

(or somehow rephrased)

Reply:
@bkeepers I just removed these details in 93df545.